.github/workflows/universalnrp-test.yml

```diff
@@ -27,8 +28,15 @@ jobs:
             { os: ubuntu, version: 22.04, package-type: DEB, tag: ''},
           ]
         arch: [amd64]
-        install-osconfig: [false]
-        policy-package: [LinuxSshServerSecurityBaselinePolicy.zip]
+        install-osconfig: [true, false]
+        policy-package: [LinuxSshServerSecurityBaselinePolicy.zip, LinuxSecurityBaselinePolicy.zip]
```

Comment:
nit: consider moving the `install-osconfig` flags inside the `policy-package` (similar to `target` above). It could help simplify the exclusions/make it clearer what combinations.

Reply:
We do not need OSConfig here at all.

Reply:
good idea, but right now we are defaulting to false, if ever we need both, ill add it within `policy-package`